sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -703,6 +703,8 @@ class SQLQueryTestSuite extends QueryTest with SharedSparkSession with SQLHelper
   }
 
   test("Test logic for determining whether a query is semantically sorted") {
+    spark.sql("CREATE SCHEMA semantic_sort_test")
```

Comment:
Wonder if we should have a simple wrapper, e.g., `SQLTestUtilsBase.withTempView`

Reply:
Please use view instead.

Reply:
@andylam-db was there any reason the test had to use tables, or can we change it to views?

Reply:
Test is working fine after changing it to temp views